config/samples/pvc_v1alpha1_datamoverbackup.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: pvc.oadp.openshift.io/v1alpha1
 kind: DataMoverBackup
 metadata:
-  name: datamoverbackup-sample
+  name: datamoverbackup-sample-2
 spec:
-  # TODO(user): Add fields here
+  volumeSnapshotContent:
+    name: snapcontent-38919783-63f3-4fe4-8c40-9e5b409e3232
```

Comment:
Should we add a field for protected ns in the example? On that note, do we need to change the API for DMB controller?

Reply:
Done, updated the PR